input/fsh/valueSets.fsh

```diff
@@ -0,0 +1,9 @@
+ValueSet: disease-codes
+Id: disease-codes
+Title: "Disease Codes"
+Description: "This value set includes all codes from NCIt (NCI Thesaurus)."
+* ^version = "0.1.0"
+* ^status = #draft
+* ^experimental = false
+* ^publisher = "NCPI Project Forge"
+* include codes from system $ncit.owl
```

Comment:
Again, curating multiple resources under a typed fsh file seems like a good idea (say valueSets, codeSystems, etc.). Previously, we have done like `ValueSet-disease-codes.json` and let's discuss what's better way + naming convention for fsh files (e.g. `valueSets` vs `valueSet` vs `value-sets` vs `value-set`). I personally prefer a sigular naming scheme.